game/inventory.h

```diff
@@ -1,43 +1,40 @@
-/*
```

Comment:
Please in the future avoid changing of line endings, indentation or anything that may produce undesirable diff. Also keep using tab indent instead of space indent because I started whole project with tab (I know about "tab vs space" holy war, but I prefer tab)

Reply:
ok sorry for it, i will change it too.

Reply:
Nah, don't worry, I'll rebase your pull request before merge with master to clean it up. Just keep tracking all whitespace diff in the future before commit.

Reply:
sure dont worry about it, i will check it before doing anything.